cmd/podman/oci-trace-hook/trace.go

```diff
@@ -0,0 +1,238 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/binary"
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	bpf "github.com/iovisor/gobpf/bcc"
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+	seccomp "github.com/seccomp/libseccomp-golang"
+)
+
+type event struct {
+	Pid uint32
+	ID  uint32
+	// Inum    uint
+	Command [16]byte
+}
+
+type calls map[string]int
```

Comment:
Is this unused? I can't find reference to it outside of here

Reply:
Wait, I see it at line 94... Can we make this a local variable, not a global?

Reply:
Still valid. Let's use `map[string]int` instead of adding a new type.